docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -289,29 +289,60 @@ <h3 id="sv"> Structural variant types</h3>
     
     <ul>
         <li><b>INS</b> - insertion</li>
+        <ul>
+          <li><b>INS:ME</b> - insertion of mobile element</li>
+          <li><b>INS:ME:ALU</b> - insertion of ALU element</li>
+          <li><b>INS:ME:HERV</b> - insertion of HERV element</li>
+          <li><b>INS:ME:LINE1</b> - insertion of LINE1 element</li>
+          <li><b>INS:ME:SVA</b> - insertion of SVA element</li>
+        </ul>
+
         <li><b>DEL</b> - deletion</li>
+        <ul>
+          <li><b>DEL:ME</b> - deletion of mobile element</li>
+          <li><b>DEL:ME:ALU</b> - deletion of ALU element</li>
+          <li><b>DEL:ME:HERV</b> - deletion of HERV element</li>
+          <li><b>DEL:ME:LINE1</b> - deletion of LINE1 element</li>
+          <li><b>DEL:ME:SVA</b> - deletion of SVA element</li>
+        </ul>
+
         <li><b>DUP</b> - duplication</li>
-        <li><b>TDUP</b> - tandem duplication</li>
+        <ul>
+          <li><b>DUP:TANDEM</b> - tandem duplication</li>
+          <li><b>TDUP</b> - tandem duplication</li>
+        </ul>
         <li><b>INV</b> - inversion</li>
+
         <li><b>CNV</b> - copy number variation</li>
         <ul>
           <li>
-            The copy number value can be specified,
-            such as <kbd>&ltCN0&gt</kbd> or <kbd>&ltCN=4&gt</kbd>
+            The copy number value can be specified like so:
+            <kbd>CN0</kbd> or <kbd>CN=4</kbd> or <kbd>CN3,CN4,CN6</kbd> or <kbd>CN=0,CN=2,CN=4</kbd> or ...
```

Comment:
Hi @nuno-agostinho , can we make them appear them as a list, it would be more readable.
And, at the eol you have `...`, is that intended?

Reply:
Converted to list and removed the `...`, thanks!